src/common/base_test.py

```diff
@@ -18,6 +18,9 @@
 
 ROOT_DIR = pathlib.Path(__file__).absolute().parent.parent.parent
 
+CHROMIUM_DIR = os.environ.get('CHROMIUM_DIR') or next((path_dirs_files[0]
```

Comment:
I have a few reservations about this:
a) the part which selects a directory by scanning ROOT_DIR should be placed in a separate function (like `non_feature_options`)
b) I'm not sure how stable the selection is, even from run to run. `os.walk` does not guarantee order of scanning. If this is intentional (i.e., use any random chromium dir), then it should be documented.

To be honest I'd suggest using some fixed default (maybe with homedir expansion, e.g., `f"{os.path.expanduser('~')}/chromium"` instead of auto-scan

Reply:
BTW: much like lines 68/69 for chromedriver

Reply:
OK, this has been fixed in a41d94d.